### PR TITLE
feat: Add endpoint for server version

### LINKED ISSUE #6470: Add server version and an API to show it
- [x] README shows wrong release information, update it to correct release
- [ ] Add an API which shows the current version of the server.

### PR DESCRIPTION
**Pull request by lladhibhutall**
Fixes #6470

#### Short description of what this resolves:
Added an API which shows the current version of the server

#### Changes proposed in this pull request:

- Added a new endpoint for server version

#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [X] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by lladhibhutall** [comment]
@prateekj117 @iamareebjamal Please review.

File: app/api/server_version.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+
+SERVER_VERSION = '1.0'
+
+server_version_route = Blueprint('server_version', __name__, url_prefix='/v1')
+
+
+class ServerVersion:
+    @staticmethod
+    @server_version_route.route('/server_version')
```
Comment (iamareebjamal) [inline app/api/server_version.py]: Make it `/info`

This JSON should be the output

```json
{
  "build": {
    "version": "2.2.3-SNAPSHOT",
    "time": "2019-10-15T07:26:58.080Z"
  }
}
```

**Comment by lladhibhutall** [comment]
I have made the changes, @iamareebjamal please review. The output json is of the following format:

`{
  "build": {
    "time": "2019-10-17 08:28:57.087928", 
    "version": "2.2.3-SNAPSHOT"
  }
}`

File: app/api/server_version.py [hunk 2]
```diff
@@ -1,13 +1,15 @@
 from flask import jsonify, Blueprint
+from datetime import datetime
 
+SERVER_VERSION = '2.2.3-SNAPSHOT'
```
Comment (iamareebjamal) [inline app/api/server_version.py]: It's not 2.2.3-SNAPSHOT

That was just an example

File: app/api/server_version.py [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
 
-SERVER_VERSION = '1.0'
-
-server_version_route = Blueprint('server_version', __name__, url_prefix='/v1')
+info_route = Blueprint('info', __name__, url_prefix='/v1')
+_build = {'time': str(datetime.now()), 'version': SERVER_VERSION}
```
Comment (iamareebjamal) [inline app/api/server_version.py]: time should be the time of release

Comment (iamareebjamal) [reply app/api/server_version.py]: Since it'll be a little bit complex for you, ignore the time field

File: app/api/server_version.py (hunk 2 above)
Comment (lladhibhutall) [reply app/api/server_version.py]: Let me know what is the version I should put.

File: app/api/server_version.py (hunk 3 above)
Comment (lladhibhutall) [reply app/api/server_version.py]: @iamareebjamal sure no issues. Although if you give me some hint about the vx pipeline I can try to figure out something.

File: app/api/server_version.py (hunk 2 above)
Comment (iamareebjamal) [reply app/api/server_version.py]: See README

File: app/api/server_version.py (hunk 3 above)
Comment (iamareebjamal) [reply app/api/server_version.py]: vx pipeline?

Comment (iamareebjamal) [reply app/api/server_version.py]: Simplest solution is to invoke git to find last commit time

Comment (iamareebjamal) [reply app/api/server_version.py]: But we need to finalize the release today, so let it be and do it in a future release

File: app/__init__.py [hunk 4]
```diff
@@ -48,6 +48,7 @@
 from app.views.redis_store import redis_store
 from app.views.celery_ import celery
 from app.templates.flask_ext.jinja.filters import init_filters
+from app.api.server_version import ServerVersion
```
Comment (iamareebjamal) [inline app/__init__.py]:
```suggestion
```

File: app/api/server_version.py [hunk 5]
```diff
@@ -0,0 +1,15 @@
+from flask import jsonify, Blueprint
+from datetime import datetime
+
+SERVER_VERSION = '2.2.3-SNAPSHOT'
```
Comment (iamareebjamal) [inline app/api/server_version.py]:
```suggestion
SERVER_VERSION = '1.7.0-SNAPSHOT'
```

File: app/api/server_version.py [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+info_route = Blueprint('info', __name__, url_prefix='/v1')
+_build = {'time': str(datetime.now()), 'version': SERVER_VERSION}
```
Comment (iamareebjamal) [inline app/api/server_version.py]:
```suggestion
_build = {'version': SERVER_VERSION}
```

File: app/api/server_version.py [hunk 7]
```diff
@@ -0,0 +1,15 @@
+from flask import jsonify, Blueprint
+from datetime import datetime
```
Comment (iamareebjamal) [inline app/api/server_version.py]:
```suggestion
```

File: app/api/server_version.py [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+
+SERVER_VERSION = '2.2.3-SNAPSHOT'
+
+info_route = Blueprint('info', __name__, url_prefix='/v1')
+_build = {'time': str(datetime.now()), 'version': SERVER_VERSION}
+
+
+class ServerVersion:
```
Comment (iamareebjamal) [inline app/api/server_version.py]: No need for a class, just define the method

File: app/__init__.py [hunk 9]
```diff
@@ -133,6 +133,7 @@ def create_app():
 
     # development api
     with app.app_context():
+        from app.api.server_version import info_route
```
Comment (iamareebjamal) [inline app/__init__.py]: Move to last

File: app/__init__.py [hunk 10]
```diff
@@ -167,6 +167,8 @@ def create_app():
         app.register_blueprint(admin_blueprint)
         app.register_blueprint(alipay_blueprint)
         app.register_blueprint(admin_misc_routes)
+        app.register_blueprint(import_routes)
```
Comment (iamareebjamal) [inline app/__init__.py]: Revert this to where it was

File: app/__init__.py [hunk 11]
```diff
@@ -149,15 +149,16 @@ def create_app():
...
 
         app.register_blueprint(api_v1)
         app.register_blueprint(event_copy)
         app.register_blueprint(upload_routes)
         app.register_blueprint(export_routes)
-        app.register_blueprint(import_routes)
         app.register_blueprint(celery_routes)
         app.register_blueprint(auth_routes)
         app.register_blueprint(event_statistics)
+        app.register_blueprint(info_route)
```
Comment (iamareebjamal) [inline app/__init__.py]: Still at the wrong place

Comment (lladhibhutall) [reply app/__init__.py]: I have moved the `/info_route` and `/import_route` to its original place. Do you want me to put `/info_route` end?

Comment (iamareebjamal) [reply app/__init__.py]: Yes

Comment (iamareebjamal) [reply app/__init__.py]: info_route had no original place as it's a new route and should be placed in the end

**Comment by lladhibhutall** [comment]
@iamareebjamal @prateekj117 This is the first Open Source PR I have ever done in my life. Thanks guys for being so patient. You guys are awesome.

**Pull request merged**